docker/Dockerfile.tests

```diff
@@ -27,7 +27,9 @@ RUN pip install .
 CMD [ "python", "--version"]
 
 FROM dev as test
-RUN pip install --no-cache-dir --requirement reqs/requirements-dev-test.txt --constraint constraints-dev.txt
```

Comment:
I've also updated the docker file to pass in all the requirements files at each step. Before we would only add the new installs and without the full requirements file, we could end up with incompatible dependency versions.